src/utils/products-filter-functions.js

```diff
@@ -0,0 +1,51 @@
+const getFilteredByCategory = (products, filterByCategory) => {
+  if (filterByCategory.length > 0) {
+    return products.filter((product) =>
+      filterByCategory.includes(product.category)
+    );
+  } else {
```

Comment:
I think there is no need to write else part it will directly get returned if `if` condition is false

Reply:
It will return undefined Kajal that's why I have put an explicit else block.